tests/integration/recipe/test_check.py

```diff
@@ -274,27 +274,6 @@ def test_valid_time_selection_rejections(timerange, message):
     assert str(rec_err.value) == message
 
 
-def test_differing_timeranges(caplog):
-    timeranges = set()
-    timeranges.add("1950/1951")
-    timeranges.add("1950/1952")
-    required_variables = [
-        {"short_name": "rsdscs", "timerange": "1950/1951"},
-        {"short_name": "rsuscs", "timerange": "1950/1952"},
-    ]
-    with pytest.raises(ValueError) as exc:
-        check.differing_timeranges(timeranges, required_variables)
-    expected_log = (
-        f"Differing timeranges with values {timeranges} "
-        "found for required variables "
-        "[{'short_name': 'rsdscs', 'timerange': '1950/1951'}, "
-        "{'short_name': 'rsuscs', 'timerange': '1950/1952'}]. "
-        "Set `timerange` to a common value."
-    )
-
-    assert expected_log in str(exc.value)
```

Comment:
can we pop in a similar test that uses the new functionality?

Reply:
I would argue that this is an integration test 😄 It tests many different methods (`from_files` calls many subfunctions) on real (temporary) files. It might be located in the wrong test module though 😅

Reply:
indeed! It's a lot more of an integration test than that integration test ever was - perfect then :smiley: